dockstore-webservice/src/main/java/io/dockstore/webservice/jdbi/EntryDAO.java

```diff
@@ -29,6 +29,11 @@
  */
 public class EntryDAO<T extends Entry> extends AbstractDAO<T> {
 
+    public final int firstIndex = 0;
```

Comment:
More descriptive names would be useful here. Failing that, comments. 
But descriptive variable names would be preferred